app/helpers/ems_infra_helper.rb

```diff
@@ -1,3 +1,10 @@
 module EmsInfraHelper
   include_concern 'TextualSummary'
+
+  def scaling_field_label(param_name)
+    field_name = param_name.dup
+    field_name.sub!("Count", "")
+    field_name.sub!("::count", "")
+    "Number of #{field_name.underscore.humanize.titleize} Hosts"
```

Comment:
I see this is not i18n. Should we i18n at least line 8?
Ping @mzazrivec

Reply:
That's one thing I wondered about, but it wasn't clear to me what's normally done here when the text is dynamic like this -- would we just translate the static part and just interpolate the dynamic text?

Reply:
For OpenStack it's a finite list. I imagine it's similar for any other providers that use this screen. While this would resolve the translation problem, it would introduce a couple of potential downsides:
1) It introduces provider-specific text in the UI (Ceph, Compute, Controller, etc.) -- I know that in the past there has been some resistance to this, but perhaps if it's only strings and not classnames or other logic it's fine here.
2) Other provider teams would need to review this as well to see what field_name values will be needed for their own providers

In any case, I imagine if we do this we'll still want to use the current automated generation of field name from this PR as a fallback for any field name values that are not found in the list. With the fallback in place, I can ignore other providers and just include the OpenStack fields, so nothing breaks for other use cases -- then, in subsequent PRs the other provider teams can supply any missing fields.

Reply:
@mzazrivec Hi, any thoughts on @sseago's comment?